src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const clone = {};
+
+  for (const key in robot) {
+    const newKey = robot[key];
+
+    if (clone.hasOwnProperty(newKey)) {
```

Comment:
Instead of using 'hasOwnProperty', it's recommended to use 'Object.prototype.hasOwnProperty.call' to avoid issues if the 'clone' object has a property with the name 'hasOwnProperty'.